spec/lib/vmdb/loggers_spec.rb

```diff
@@ -1,5 +1,12 @@
+require "stringio"
+
 RSpec.describe Vmdb::Loggers do
-  let(:log_file) { Rails.root.join("log", "foo.log").to_s }
+  let(:log_file_name) { "foo.log" }
+  let(:log_file_path) { Rails.root.join("log", log_file_name) }
+
+  after do
+    log_file_path.delete if log_file_path.exist?
+  end
```

Comment:
This might cause issues with parallel specs, but I'm not sure.

Reply:
yeah, I think we need to use `Dir.mktmpdir` or `Tempfile.new.path` to ensure any two examples from this context running at the same time won't stomp on each other's log files.

Reply:
I couldn't use a tempdir or tempfile because this is testing the create_logger interface which creates the logs in the logs dir.  Instead changed the filename to `"#{SecureRandom.alphanumeric}.log"`, which should be sufficient if the tests are paralellized.